internal/lambdas/api/get_comments.go

```diff
@@ -0,0 +1,178 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/samber/lo"
+
+	"github.com/salt-today/salttoday2/internal/sdk"
+	"github.com/salt-today/salttoday2/internal/store"
+)
+
+var somethingWentWrong []byte = []byte("Something went wrong")
+
+func GetCommentsHTTPHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
Talked to @Clairev3 and @Nexushunter and they seemed in favour of a general api/v1/comments endpoint since they're going to be allowing users to set a bunch of filter options which they can then just add as query params to this endpoint